lib/internal/Magento/Framework/MessageQueue/Topology/Config/Data.php

```diff
@@ -23,4 +23,20 @@ public function __construct(
     ) {
         parent::__construct($reader, $cache, $cacheId, $serializer);
     }
+
+    public function getQueues(): array {
```

Comment:
Since `->get()` loses backward compatibility (it returns both `exchanges` and `queues` when it used to only return `exchanges`), how should we consider the compatibility of this API?

Reply:
cc: @paliarush